web/src/app/util/Search.js

```diff
@@ -51,22 +84,34 @@ export default function Search() {
     return (
       <TextField
         InputProps={{
-          disableUnderline: true,
-          classes: {
-            input: classes.searchFieldBox,
-          },
+          ref: fieldRef,
+          startAdornment: (
+            <InputAdornment position='start'>
+              <SearchIcon color='action' />
+            </InputAdornment>
+          ),
+          endAdornment: props.endAdornment && (
+            <InputAdornment position='end'>
+              {React.cloneElement(props.endAdornment, { anchorRef: fieldRef })}
+            </InputAdornment>
+          ),
         }}
+        data-cy='search-field'
         placeholder='Search'
+        variant='outlined'
+        margin='dense'
+        hiddenLabel
         onChange={e => setSearch(e.target.value)}
         value={search}
+        className={textClass}
         {...extraProps}
       />
     )
   }
 
   function renderMobile() {
     return (
-      <React.Fragment>
+      <PageActions>
```

Comment:
Does `PageActions` support being on the page multiple times/places?

Reply:
It does not